msbuild/Xamarin.Mac.Tasks.Core/Tasks/MmpTaskBase.cs

```diff
@@ -237,9 +233,14 @@ protected override string GenerateCommandLineCommands ()
 				Log.LogWarning ("Failed to create response file '{0}': {1}", responseFile, ex);
 			}
 
-			// Use only the response file
+			// Reponse file must come before extra arguments
```

Comment:
minor: that's saying what's done (kind of obvious) but not why (which is less obvious)
don't update unless something else needs fixing (blame / git history will tell the story)